config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  resources :messages
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
+
+  # Defines the root path route ("/")
+  root "messages#index"
+end
```

Comment:
- [ ] When I try to access the greetings from the browser, what is rendered is the **_JSON_** format of the random message from the database. The screenshot below would explain this better 🖱️:

![image](https://github.com/waelelsafty07/hello-rails/assets/77106962/bdb24f1e-1850-4587-9513-40888f5fca8d)

---

The greeting should be rendered via the frontend through the React application as HTML and not as JSON. I would advise you to not use the `Messages#index` as the root route but to create a static controller and view and make it the root. Please ensure that the application works properly and messages are displayed via the frontend that you have configured. 😎 💻